articles/introducing-cross-platform-script-execution.md

```diff
@@ -0,0 +1,61 @@
+
+
+### Why cross-platform script execution matters
+
+In the rapidly evolving world of IT, security, and device management, the demands placed on administrators and security engineers are continually increasing. The challenges of keeping systems up-to-date, secure, and efficiently managed require tools that are both powerful and adaptable.
+
+The introduction of the cross-platform script execution feature in Fleet is a direct response to these needs. But why does this matter, and what makes this new capability so essential?
+
+
+
+* **Enhanced efficiency and flexibility:** Running a script across multiple operating systems – macOS, Windows, and Linux – means that administrators no longer need to navigate different tools for different systems. Whether you are remediating an issue or collecting logs, this feature streamlines the process, saving time and effort.
+* **Root-level security control:** The ability to execute scripts is not just handed out freely. It requires root access to the device, reflecting a carefully considered approach to security. This access level ensures that the powerful capabilities are matched with stringent safeguards, only enabling those with the proper permissions to utilize this feature.
+* **Practical application in real-time remediation and investigation:** In a world where immediate response is often critical, the ability to run a script one-off per host provides IT professionals with the tools to act swiftly. Whether addressing vulnerabilities, conducting inquiries, or implementing standard compliance policies, this feature provides a practical solution tailored to the realities of today's technology landscape.
+* **Alignment with Fleet's core values**: At Fleet, we are dedicated to making security and IT interoperable and easy to automate. This feature aligns with our commitment to clarity and trust through open-source software and fits seamlessly with our values of empathy, ownership, results, objectivity, and openness.
+
+
+### Supported platforms
+
+Fleet's new script execution supports macOS, Windows, and Linux systems. For macOS and Linux, the execution will utilize the Bourne (`sh`) shell. On Windows, PowerShell scripts are supported, providing a flexible and robust range of options for IT administrators and security engineers.
+
+
+### Prerequisites
+
+Before leveraging the power of this feature, specific prerequisites must be met. Most crucially, it can only be enabled by someone with root access to the device. This access can be proven by pushing a new package, modifying a configuration file with root-only write access, or pushing an MDM profile. Fleet requires this level of access to provide a more convenient mechanism for script execution.
+
+For macOS and Linux, the script execution relies on shell scripts specified in the shebang interpreter. Therefore, it's essential to ensure the chosen interpreter, such as sh or zsh, is available and configured correctly on the target device.
```

Comment:
```suggestion
For macOS and Linux, the script is always executed in the device's (root) default shell (`/bin/sh`). Therefore, it's important to understand which shell is configured as the default on the target device.
```

I think we need to clarify that scripts run in the default shell. You can't specify different interpreters.